packages/headless/src/api/commerce/product-listings/product-listing-params.ts

```diff
@@ -34,6 +38,53 @@ export interface ProductListingRequestParam {
   userContext?: ContextPayload;
 }
 
+// TODO(nico): Use interfaces for nested params
+export interface ProductListingV2RequestParam {
```

Comment:
Yeah, and I think for some of those parameters they are already defined in the project... 🤔

There will be some overlap with what's defined in `api/search` for example, which we can probably just refactor/tweak a bit to reuse in both places.